gravity-forms/gw-cache-buster.php

```diff
@@ -190,10 +191,15 @@ public function is_cache_busting_applicable() {
 
 	public function suppress_default_post_render_event( $form_string, $form, $current_page ) {
 
-		$footer_script_body = "gform.initializeOnLoaded( function() { jQuery(document).trigger('gform_post_render', [{$form['id']}, {$current_page}]) } );";
-		$search             = GFCommon::get_inline_script_tag( $footer_script_body );
+		$searches = array(
+			"gform.initializeOnLoaded( function() { jQuery(document).trigger('gform_post_render', [{$form['id']}, {$current_page}]) } );",
```

Comment:
It would be sweet if we could use a Regex for this, but it gets tricky to do so.

This is close, but due to the wildcards towards the end, it matches *too* much of the form string.

```
gform\.initializeOnLoaded\(\s*function\(\)\s*{[\s\S]*?jQuery\(document\)\.trigger\('gform_post_render',[\s\S]\[[0-9]+, [0-9]+\]\)[\s\S]*?}\s*\);
```

Reply:
Per our Slack convo, probably not worth the trickery to make this happen.